src/gigs-board/entity/post/PostEditor.jsx

```diff
@@ -110,15 +113,9 @@ if (!state.draftStateApplied && props.draftState) {
 let fields = {
   Comment: ["description"],
   Idea: ["name", "description"],
-  Submission: ["name", "description"],
+  Submission: ["name", "description", "fund_raising"],
   Attestation: ["name", "description"],
-  Sponsorship: [
-    "name",
-    "description",
-    "amount",
-    "sponsorship_token",
-    "supervisor",
-  ],
+  Sponsorship: ["name", "description", "fund_raising"],
```

Comment:
Is this intentional? I saw you proposed to remove this in the issue, but @ori-near hasn't reply so is this decided.

Reply:
Yes I spoke with @ori-near on a call about this.